### DISCUSSION TITLE
FCC trace problem

### DISCUSSION THREAD

**Discussion by WongHsai** [mtex-toolbox/mtex#1286] [Ask Anything]
Hello Community!
After definied slip systems to plot specific trace of grain of interest(GOI),none of the four orientations of ploted lines match the trace line obtained from the script from mtex Schmid section.After import_wizard&smooth section,My code is like:
```matlab
cs=ebsdhq.CS
%%four slip planes are required
sS1=slipSystem(Miller(-1,1,0,cs,'UVW'),Miller(1,1,1,cs,'HKL'));
sS4=slipSystem(Miller(-1,-1,0,cs,'UVW'),Miller(-1,1,1,cs,'HKL'));
sS7=slipSystem(Miller(1,-1,0,cs,'UVW'),Miller(-1,-1,1,cs,'HKL'));
sS10=slipSystem(Miller(1,1,0,cs,'UVW'),Miller(1,-1,1,cs,'HKL'));
sSLocal1=grainsGOI.meanOrientation*sS(1)
 sSLocal4=grainsGOI.meanOrientation*sS(4)
 sSLocal7=grainsGOI.meanOrientation*sS(7)
 sSLocal10=grainsGOI.meanOrientation*sS(10)
quiver(grainsGOI,sSLocal1.trace,'color','red')
hold on
quiver(grainsGOI,sSLocal4.trace,'color','blue')
hold on
quiver(grainsGOI,sSLocal7.trace,'color','green')
hold on
quiver(grainsGOI,sSLocal10.trace,'color','pink')
hold off
```
It comes like:
![捕获2](https://user-images.githubusercontent.com/90515523/143469670-480d81d0-8dad-471c-a72b-a3cf39821d2f.JPG)

Despite the mismatch problem,I plot the other three planes through method on mtex doc,
![捕获](https://user-images.githubusercontent.com/90515523/143469396-91aa4f17-746f-4bf0-9c55-d3b25284bc65.JPG)
It seems there exists a mirror like relationship between orientations,despite the color mismatch.I believe it is a slip system definition problem.Can anyone help to find out where the real problem exists?
Cheers!
Wong Hsai

**Comment by eugenianietov** [comment]
Hi! I was doing the same task and ended up here. The right way to get the sSLocal slip systems would be:

sSLocal1=grainsGOI.meanOrientation * sS1;

That way you get the right traces. Hope it helps somebody else :)

Eugenia
